packages/react-components/react-tabster/src/hooks/useListenFocusedElement.ts

```diff
@@ -0,0 +1,20 @@
+import type { Types as TabsterTypes } from 'tabster';
+import { useIsomorphicLayoutEffect } from '@fluentui/react-utilities';
+
+import { useTabster } from './useTabster';
+
+/**
+ * Subscribes to the tabster focused element. Calls the callback when the focused element changes.
+ * @param callback - Callback to subscribe to the focused element.
+ */
+export function useListenFocusedElement(
+  callback: TabsterTypes.SubscribableCallback<HTMLElement | undefined, TabsterTypes.FocusedElementDetail>,
+) {
+  const tabster = useTabster();
+  useIsomorphicLayoutEffect(() => {
```

Comment:
This can be just `useEffect()`, right? We don't need it to be done before DOM paint